### DISCUSSION TITLE
Mac OS always flags as Malware after each update (Brew)

### DISCUSSION THREAD

**Discussion by OfficePossum** [Eloston/ungoogled-chromium#2114] [Q&A] [answered]
Hello,
So i have an issue regarding updates on chromium. I basically update via the brew package manager on my Mac, although the system keeps making me go to the security and privacy section of the settings to allow for it to run.
Is there a work around someone here might of figured out? or is there a better thing to use than brew in this case?

**Comment by networkException** [comment, marked as answer]
I assume avoiding those verification steps without disabling macOS security features outright (if that's even possible) would require us to sign the binary which is probably expensive and definitely non trivial

**Reply by OfficePossum** [reply]
Makes complete sense tbh. even trying to work around it would be too much as well. thanks for maintaining the project though :)
